packages/ui-date-time-input/src/DateTimeInput/README.md

```diff
@@ -147,6 +147,47 @@ class Example extends React.Component {
 render(<Example />)
 ```
 
+A `DateTimeInput` with some disabled dates and a custom error message for them:
```

Comment:
Could you please add an example for the `string[]` version as well?

Also, now the page has a few examples, I would suggest adding subtitles for the examples so that it can be seen easier what they are showcasing and the new ToC will render on them too.